frontends/web/test/components/message/message.test.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h } from 'preact';
+import { shallow, deep } from 'preact-render-spy';
+
+import Message from '../../../src/components/message/message';
+
+describe('components/message/message', () => {
+  it('should use type attr as CSS class', () => {
```

Comment:
Doesn't this indentation give you lint errors?

Reply:
nope. `make weblint` didn't complain. I guess it's because it only runs for `frontends/web/src/`. I guess I can add `frontends/web/test/` to the lint as well?

Reply:
done

Reply:
I meant: my test files to use 4 spaces indent.